v3/tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "strict": true,
     "types": [],
     "jsx": "react",
-    "lib": ["dom", "dom.iterable", "es2018"],
+    "lib": ["dom", "dom.iterable", "es2019"],
```

Comment:
At this point my inclination would be to set `target` and `lib` to `es2020`, which is five years ago at this point, although I'd understand wanting to be conservative in this PR.

Reply:
I'll leave it alone in this PR, but now that it is updated in main, I'll try to remember to update it when we come back to this.